src/blocks/remote-data-template/block.json

```diff
@@ -4,7 +4,7 @@
   "name": "remote-data-blocks/template",
   "version": "0.1.0",
   "usesContext": [ "remote-data-blocks/remoteData" ],
-  "title": "Remote Data Template",
+  "title": "Item Template",
```

Comment:
@smithjw1 Is "item" the correct noun? What should we generically call representations of remote data entities?

Reply:
Sounds good then! 👍